cmd/envtool/envtool.go

```diff
@@ -401,6 +401,12 @@ var cli struct {
 		} `cmd:"" help:"read files"`
 	} `cmd:""`
 	PackageVersion struct{} `cmd:"" help:"Print package version"`
+	Tests          struct {
+		Shard struct {
+			Index uint `help:"Iteration's index" required:""`
+			Total uint `help:"Total number of integration tests to be listed" required:""`
```

Comment:
That should be the total number of shards. For example, `-index=2 -total=5` means there are 5 shards with an index from 0 to 4, and we want to run a shard with an index 2.

We also should check somewhere that `index < total`